.php-version

```diff
@@ -0,0 +1 @@
+8.2
```

Comment:
This affects the Symfony binary, so if we in the future use that in the build it could have unintended side effects.